lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -0,0 +1,158 @@
+# encoding: utf-8
+require "logstash/outputs/base"
+require "logstash/namespace"
+require "march_hare"
+require "java"
+
+# Common functionality for the rabbitmq input/output
+module LogStash::PluginMixins::RabbitMQConnection
```

Comment:
Sure thing Guy, I'll switch to the nested version :)

On Thu, Aug 20, 2015 at 5:19 PM, Guy Boertje <notifications@github.com>
wrote:

> In lib/logstash/plugin_mixins/rabbitmq_connection.rb
> <https://github.com/logstash-plugins/logstash-mixin-rabbitmq_connection/pull/1#discussion_r37587751>
> :
>
> > @@ -0,0 +1,158 @@
> > +# encoding: utf-8
> > +require "logstash/outputs/base"
> > +require "logstash/namespace"
> > +require "march_hare"
> > +require "java"
> > +
> > +# Common functionality for the rabbitmq input/output
> > +module LogStash::PluginMixins::RabbitMQConnection
>
> (pleading) lets not perpetuate this style of declaration ->
> ```ruby module LogStash module PluginMixins module RabbitMQConnection
> ....
> end end end
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/logstash-plugins/logstash-mixin-rabbitmq_connection/pull/1/files#r37587751>
> .
>